CMakeLists.txt

```diff
@@ -582,7 +582,7 @@ if (BUILD_TESTING)
                 add_test(NAME ${test_target}
                         COMMAND
                         pytest
-                        -x -n=${N} --maxfail=1 --reruns=0 --cache-clear -rpfsq
```

Comment:
Is there any benefit to keeping `maxfail`? I vaguely remember it being desirable for cases where a change causes the integration tests to fail due to timeout, which makes running each test very slow and causes the entire codebuild job to timeout.

Reply:
`-x` should cover this:  ` -x, --exitfirst       Exit instantly on first error or failed test` - maxfail should really require a value >=2.